package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "wtc-gallery-component",
-  "version": "1.4.2",
+  "version": "1.4.3",
```

Comment:
Let's change this to a minor version update instead of a patch.

Reply:
Updated, thanks!

Reply:
Sorry, versions cascade. So if you're updating the minor version, the patch version should become `0`

Reply:
Cool - good to know. Updated.